tests/test_nc_to_mmd.py

```diff
@@ -532,6 +533,32 @@ def test_json_dry_run(dataDir):
     tmp = Nc_to_mmd(test_json_header, json_input=True, check_only=True)
     tmp.to_mmd()
 
+    with monkeypatch.context() as mp:
+        mp.setattr(nc_wrapper, "__init__", lambda *a, **k: None)
+        mp.setattr(nc_wrapper, "ncattrs", lambda *a, **k: ["what", "ever"])
+        mp.setattr(nc_wrapper, "getncattr", lambda *a, **k: ["what", "ever"])
+        with pytest.raises(ValueError) as ee:
+            tmp = Nc_to_mmd(test_json_header, json_input=True,
+                            opendap_url="https://thredds.met.no/etc", output_file="somefn.xml")
+        assert "Input parameter 'file_size' must be " in str(ee.value)
+
+    with pytest.raises(ValueError) as ee:
+        tmp = Nc_to_mmd(test_json_header, json_input=True, file_size=10,
+                        opendap_url="https://thredds.met.no/etc", output_file="somefn.xml")
+    assert "Input parameter 'target_nc_filename' must be " in str(ee.value)
```

Comment:
Error message are not functional, I don't think updating error messages should break unittesting.

Reply:
Not sure about this. First, I don't think it is very important, as it is easy to update the test. Second, I can't think of another alternative right away. Please suggest a change if you insist.

Reply:
I think it should be sufficient to just test that the correct error is raised. No need to check the content of the error.